lib/fog/compute/kubevirt/requests/create_vm.rb

```diff
@@ -4,15 +4,14 @@ class Kubevirt
       class Real
         def create_vm(vm)
           kubevirt_client.create_virtual_machine(vm)
-        rescue Kubeclient::HttpError
+        rescue KubeException => err
```

Comment:
I thought that the client would wrap it in `Kubeclient::HttpError`. If it is not the case it seems like the client bug.

Reply:
I'll revert back.

Reply:
that was probably a merge issue since I haven't edited those line myself. We shouldn't use KubeException anymore.